src/fileimpl.cpp

```diff
@@ -385,6 +385,16 @@ namespace zim
     return ret;
   }
 
+  std::shared_ptr<const Cluster> FileImpl::getClusterFromDisk(cluster_index_t idx)
```

Comment:
I would call it `createCluster` or `readCluster`.

We are not especially get the cluster from the disk (at least, not more than `getCluster`).
We have no real concept of "Disk". Having it in only one function name seems a bit odd.

Reply:
Done